community/kernel/src/main/java/org/neo4j/kernel/impl/store/id/validation/IdValidator.java

```diff
@@ -88,4 +88,10 @@ public static void assertIdWithinCapacity( long id, long maxId )
             throw new IdCapacityExceededException( id, maxId );
         }
     }
+
+    public static boolean hasReservedIdInRange( long startIdInclusive, long endIdExclusive )
```

Comment:
No, these gaps will not be in the freelist. Doesn't matter much though as it's in the size of thousands of ids.